promptsource/templates/GEM/mlsum/de/templates.yaml

```diff
@@ -0,0 +1,86 @@
+dataset: GEM/mlsum
+subset: de
+templates:
+  104b2645-9b2a-4d08-8885-3de18af246fc: !Template
+    answer_choices: null
+    id: 104b2645-9b2a-4d08-8885-3de18af246fc
+    jinja: "My college roommate asked me what this German article meant:\n {{text}}\n\
+      So I recapped it in layman''s terms in German: ||| {{target}}"
+    metadata: !TemplateMetadata
+      choices_in_prompt: false
+      metrics:
+      - ROUGE
+      - BLEU
+      original_task: true
+    name: layman_summ_de
+    reference: http://gptprompts.wikidot.com/prompt:summarization
+  6ffadb8a-c670-4d6c-97fd-9eea35945452: !Template
+    answer_choices: null
+    id: 6ffadb8a-c670-4d6c-97fd-9eea35945452
+    jinja: "{{text}}\n TL;DR in German: ||| {{target}}"
+    metadata: !TemplateMetadata
+      choices_in_prompt: false
+      metrics:
+      - ROUGE
+      - BLEU
+      original_task: true
+    name: tl_dr_de
+    reference: ''
+  a4f6a6c1-ce10-463b-932e-41a9336c3ecf: !Template
+    answer_choices: null
+    id: a4f6a6c1-ce10-463b-932e-41a9336c3ecf
+    jinja: "{{text}}\n      ===\nWrite a summary of the text above : ||| {{target}}"
```

Comment:
it seems that some of these use actual linebreaks and some use \n to signal them. For consistency sake, should we use the same thing for every prompt?

Reply:
Will fix it!

Reply:
Upon further review, this was something that was done by promptsource when making the prompts through the UI